google/ggrc-core
# GGRC-8809 Unexpected GET request is fired an all objects except Scope objects

# Issue description

All page data received from the first GET request (document type) as GGRC.page_object
After info page opened, app fires additional GET request and gets same data from this.
Reproducible only for objects with CaUpdate Mixin e.g. Audit/Control/Workflow etc.

# Steps to test the changes

For clarify what is unexpected request check ticket please. 
1.  Open GGRC app
2. Open ChromeDev tools => Network tab
3. Open any object e.g Control
4. Expected: Additional GET request doesn't fire.
5. Add all types of CA on some object from Administration panel
6. Open info pane of modified object and review that CA initialized with correct types
7. Expected: Types is correct.

# Solution description

Deleted outdated logic in CaUpdate Mixin. In previously some CA initialized with errors and had wrong types. After some investigation, I didn't saw any troubles with this. What i mean is we don't need to refresh instance anymore.

# Sanity checklist




# PR Review checklist
